### ISSUE TITLE
aligning with BIDS

### ISSUE THREAD

**Issue by CPernet** [spinalcordtoolbox/PAM50#11]
Hey guys,

I'm just putting the [link here](https://docs.google.com/document/d/1ugBdUF6dhElXdj3u9vw0iWjE6f_Bibsro3ah7sRV0GA/edit?usp=sharing) about the BIDS atlas BEP. This would be fantastic if you could contribute.

From the purely atlas labeling side (https://github.com/spinalcordtoolbox/PAM50/tree/master/atlas) this should be easy, tsv and json for labels and provenance

From the more quantitative size (https://github.com/spinalcordtoolbox/PAM50/tree/master/histology) it would be interesting to see if this works ; we have proposed something for PET and that's somehow similar 

thx

**Comment by jcohenadad** [comment]
Thank you Cyril-- I know we've discussed it internally and we have identified some roadblocks that I can't remember. I will put it on our agenda for our next meeting and we will document here.

**Comment by joshuacwnewton** [comment]
I'm just going to copy over the previous discussion from the email chain, so that the information is all in one place.

----

@mguaypaq:

> I had a look at the BIDS-Atlas BEP, and now I'm trying to see if I can fit the PAM50 template into it, and already I have several questions. So far, it looks like it would require much more re-organization and transformations than just renaming some files. (This would also require changes in the SpinalCordToolbox code which uses the PAM50 template, but that's maybe a separate matter.)
> 
> Maybe you'd rather answer these questions (and my underlying confusion) more efficiently in a meeting, but here are some questions to begin with. I know you're more interested in the Quantitative aspects of PAM50, but so far I'm still looking at the "traditional" aspects:
> 
> 1. PAM50 comes with some template images (T1-, T2-, and T2*-weighted) so that subject images can be registered with it. But, BIDS-Atlas seems to only allow probseg/dseg/mask as image types. Is there some compliant way to keep the templates together with the atlas? I'll note that the templates are not really "subjects" in a meaningful way, so I don't think that's the solution.
> 
> 2. PAM50 seems to use all of probseg/dseg/mask types, with different types for some different structures (spinal cord, white matter, etc.). Is each type a different atlas? But then, we couldn't use consistent numbering across the different types; in other words, the probseg node 0 would not refer to the same structure as dseg node 0, or mask node 0.
> 
> 3. PAM50 comes with two ways of representing spinal cord levels: a discrete version (which is probably a dseg?), and a continuous version (from 1.0 to 21.0, with floating point values). What's the data type for the continuous version? It doesn't seem to be a propseg, but maybe it's a mask? And can the two notions of level co-exist in the same atlas?
> 
> 4. For the continuous levels, it seems like some of the fields for "relmat.json" files would apply to describe the range and interpretation of values, but the levels are not a relation matrix: each value corresponds to a single voxel, not a pair of nodes. Where would this fit in BIDS-Atlas?
> 
> 5. Currentlty, PAM50 is structured as a separate 3D .nii.gz file for each node, with the node number in the filename. For BIDS-Atlas, would they have to be combined as separate volumes inside a single 4D .nii.gz? (Or even 5D, if the 4th dimension is reserved for time?)

Peer:

> thank you very much Mathieu for all the comments and feedback, 
that’s highly appreciated.
> 
> I really like how the quantitative data y’all are working with is
stressing the atlas proposal concerning its usability and, in turn,
required adaptations. I was wondering if we should organize a
meeting these days to discuss and address certain points, ie
a one hour mini sprint? @ Mathieu, Julien, Agâh et al.: I might be 
in MTL late May/early June and could stop by at Poly and we 
could zoom in Cyril. Otherwise, we can of course also schedule
a fully virtual meeting. WDYT?

----

Did we ever get a chance to meet with Peer? I recall that there were some scheduling conflicts with ISMRM at the time...

**Comment by mguaypaq** [comment]
No, the meeting never happened, and then I think we all just forgot to follow up.

**Comment by jcohenadad** [comment]
Related to https://github.com/bids-standard/bids-specification/issues/1281
